src/TestFramework/PhpUnit/Coverage/CoverageXmlParser.php

```diff
@@ -75,14 +75,14 @@ private function processXmlFileCoverage(string $relativeCoverageFilePath, string
 
         $defaultCoverageFileData = ['byLine' => [], 'byMethod' => []];
 
-        if ($percentage === '0.00' || empty($percentage)) {
+        if (!$percentage) {
```

Comment:
seems like this is not identical

`'0.00'` string is not falsy, but truthy https://3v4l.org/osjdM

Reply:
Oh well. I'll see if I can add a test for this variant.

Reply:
No easy way to test this unfortunately. I seems like 261a483b is the best course of action here. WDYT?